### PR TITLE
Decoding the Path URL

### PR DESCRIPTION
**Pull request by kain-ap**
Decoding the Path URL when there is space in VirSat Diagram Title

### CODE REVIEW

File: de.dlr.sc.virsat.graphiti.ui/src/de/dlr/sc/virsat/graphiti/ui/diagram/editor/VirSatDiagramUpdateBehavior.java [hunk 1]
```diff
@@ -146,8 +150,14 @@ protected void createEditingDomain(IDiagramEditorInput input) {
 		// Dont create a new editing domain, instead get the virsat editing domain
 		// associated with the project of this file 
 		DiagramEditorInput diagramInput = (DiagramEditorInput) input;
-		Path path = new Path(diagramInput.getUri().toPlatformString(false));
-		IResource resource = ResourcesPlugin.getWorkspace().getRoot().findMember(path);
+		String decodedPath = null;
+		try {
+			decodedPath = URLDecoder.decode(diagramInput.getUri().toPlatformString(false), StandardCharsets.UTF_8.name());
+		} catch (UnsupportedEncodingException e) {
+			e.printStackTrace();
```
Comment (PhilMFischer) [inline de.dlr.sc.virsat.graphiti.ui/src/de/dlr/sc/virsat/graphiti/ui/diagram/editor/VirSatDiagramUpdateBehavior.java:157]: Please report errors via the logging service

File: de.dlr.sc.virsat.graphiti/src/de/dlr/sc/virsat/graphiti/util/DiagramHelper.java [hunk 2]
```diff
@@ -88,7 +91,14 @@ public static StructuralElementInstance getOwningStructuralElementInstance(EObje
 			seiUri = seiUri.appendFileExtension(VirSatProjectCommons.FILENAME_EXTENSION);
 			seiUri = URI.createPlatformResourceURI(seiUri.toFileString(), true);
 			
-			Path path = new Path(resourceUri.toPlatformString(false));
+	
+			String decodedPath = null;
+			try {
+				decodedPath = URLDecoder.decode(resourceUri.toPlatformString(false), StandardCharsets.UTF_8.name());
+			} catch (UnsupportedEncodingException e) {
+				e.printStackTrace();
```
Comment (PhilMFischer) [inline de.dlr.sc.virsat.graphiti/src/de/dlr/sc/virsat/graphiti/util/DiagramHelper.java:99]: Same here

**Review by PhilMFischer — CHANGES_REQUESTED** [review]

File: de.dlr.sc.virsat.graphiti.ui/src/de/dlr/sc/virsat/graphiti/ui/diagram/editor/VirSatDiagramUpdateBehavior.java (hunk 1 above)
Comment (kain-ap) [reply de.dlr.sc.virsat.graphiti.ui/src/de/dlr/sc/virsat/graphiti/ui/diagram/editor/VirSatDiagramUpdateBehavior.java]: Added the Activator for logging service

File: de.dlr.sc.virsat.graphiti/src/de/dlr/sc/virsat/graphiti/util/DiagramHelper.java (hunk 2 above)
Comment (kain-ap) [reply de.dlr.sc.virsat.graphiti/src/de/dlr/sc/virsat/graphiti/util/DiagramHelper.java]: Added the Activator for logging service here as well

File: de.dlr.sc.virsat.apps.feature/pom.xml [hunk 3]
```diff
@@ -18,7 +18,7 @@ SPDX-License-Identifier: EPL-2.0
     <relativePath>../maven/pom.xml</relativePath>
     <groupId>de.dlr.sc.virsat</groupId>
     <artifactId>de.dlr.sc.virsat.parent</artifactId>
-    <version>4.17.1-SNAPSHOT</version>
+    <version>4.17.0-SNAPSHOT</version>
```
Comment (PhilMFischer) [inline de.dlr.sc.virsat.apps.feature/pom.xml:21]: Major concern. This is reverting back all the changes from development. E.g. switching back from 4.17.1 to 4.17.0

**Review by PhilMFischer — CHANGES_REQUESTED** [review]

**Comment by PhilMFischer** [comment]
Major issue with merging back to development. Since the problem could not be identified this PR is closed and replaced by #1222.

**Pull request closed without merging**